build/packages-template/bbb-config/plugins-locales-cors.nginx

```diff
@@ -0,0 +1,16 @@
+# Allow CORS for all JSON files under the /plugins/ URL path (including subdirectories)
+location ~ ^/plugins/.+\.json$ {
+  root /var/www/bigbluebutton-default/assets;
+  try_files $uri =404;
+
+  add_header Access-Control-Allow-Origin "*" always;
+  add_header Access-Control-Allow-Methods "GET, OPTIONS" always;
+  add_header Access-Control-Allow-Headers "Origin, Content-Type, Accept" always;
+
+  if ($request_method = OPTIONS) {
+    add_header Access-Control-Max-Age 1728000;
+    add_header Content-Type "text/plain charset=UTF-8";
```

Comment:
I tested the PR for the h5p plugin and I've had some problems with some Fonts, see error below:

<img width="720" height="248" alt="image" src="https://github.com/user-attachments/assets/db382ba2-3c6e-4f23-9cbe-83e54e35cc48" />

This can be happening due to the way the font is imported (possibly via CSS link).

I don't think it's a bad idea to allow every content-type, specially in this case, when we don't really have control over the activity we are rendering from h5p.